libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/SalesforceActivity.java

```diff
@@ -38,6 +38,13 @@ public abstract class SalesforceActivity extends AppCompatActivity implements Sa
 
 	private final SalesforceActivityDelegate delegate;
 
+	/**
+	 * An option to build the REST client automatically on activity resume, which has the side
+	 * effect of displaying the default login activity when no user is logged in.
+	 */
+	@Deprecated
+	protected boolean buildRestClientOnResume = true;
```

Comment:
What exactly is this for?  The host app to set in their activity if they are using QR Code Login?  It does not appear to be used anywhere.